pkg/apis/aws/validation/worker_test.go

```diff
@@ -131,11 +137,18 @@ var _ = Describe("ValidateWorkerConfig", func() {
 		})
 
 		It("should return no errors for a valid gp2 configuration", func() {
-			worker.Volume.IOPS = &gp2iops
-			Expect(ValidateWorkerConfig(worker, &core.Volume{Type: &gp2type}, dataVolumes, fldPath)).To(BeEmpty())
+			worker.Volume.IOPS = nil
```

Comment:
add the case of providing gp2iops also passing , but comment that it would fail later by AWS as currently gp2 doesn't support iops.